examples/embed-map/index.html

```diff
@@ -0,0 +1,11 @@
+<html>
+  <body>
+    <h1> Icelab in Canberra </h2>
+    <div data-view-embed-map='{"cityCoordinates": {"lat": -35.279115, "lng": 149.126466},"coordinates": {"lat": -35.280937,"lng": 149.130009}}'></div>
+    <h1>Icelab in Melbourne</h1>
+    <div data-view-embed-map='{"cityCoordinates": {"lat": -37.8136, "lng": 144.9631},"coordinates": {"lat": -37.7971762, "lng": 144.9855213}}'></div>
+
+    <script src="./bundle.js"></script>
```

Comment:
Related to:

> Do we need to attached this to an event listener? I would have thought it’d work fine without it?

I think the order of these scripts should be reversed so we load the JS synchronously and just execute it immediately.